ecl2df/summary.py

```diff
@@ -287,6 +287,72 @@ def df(
     return dframe
 
 
+def _fix_dframe_for_libecl(dframe: pd.DataFrame) -> pd.DataFrame:
+    """Fix a dataframe making it ready for EclSum.from_pandas()
+
+    * Ensures that the index is always datetime, and sorted.
+    * Removes BLOCK vectors, these are currently not supported as
+      it requires knowledge of the grid dimensions. Warnings
+      will be emitted for skipped columns
+
+    Args:
+        dframe (pd.DataFrame): Dataframe to read. Will not be modified.
+
+    Returns:
+        pd.DataFrame: Modified copy of incoming dataframe.
+    """
+    dframe = dframe.copy()
+    if "DATE" in dframe.columns:
```

Comment:
Make this case insensitive? E.g allow "Date", "date" ...?

Reply:
And move "Date" out of method, defined as a constant DATAFRAME_DATE_COLUMN_NAME="DATE" og deretter sjekke på feks `if DATAFRAME_DATE_COLUMN_NAME in [column.upper() for column in dframe.columns]` ?

Reply:
I am quite unsure to which degree we should support different variations of header names. All other columns are required to be exact on case and letters, and also for the other ecl2df modules.